languages/purescript/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,58 @@
+PureScript is a statically-typed language, meaning that the type of a variable is checked when the program is compiled. Using the `=` operator, we can bind a value to a variable name:
+
+```purescript
+variable = 10 -- Bind the integer value 10
+```
+
+The value of a variable can not change once defined. Instead, a new variable can be created:
+
+```purescript
+variableOne = 1 -- Bind an integer value of 1 to variableOne
+variableTwo = 2 -- Bind an integer value of 2 to variableTwo
+
+-- The compiler will output an error when trying to re-bind the value of a variable:
+variableTwo = 3
+-- => Multiple value declarations exist for variableTwo.
+```
+
+The type of `variableOne` can be said to be an integer (known as an `Int` in PureScript). You can declare the type of a variable before declaring the variable, for example:
+
+```purescript
+variableOne :: Int -- variableOne is now of type Int. This syntax is known as the type signature declaration.
+variableOne = 1 -- Binds the value of 1 to the Int variableOne
+```
+
+However, type can often be inferred by the compiler. **Note**: The `::` can be read "has type".
+
+PureScript is an [functional-programming language][functional-programming] and requires all named functions to be defined in a [module][modules]. The `module` keyword is used to define a module. All modules are available to all other modules at runtime. A [module][modules] is analogous to a _class_ in other programming languages.
+
+```purescript
+module Calculator where
+-- ...
+```
+
+Functions must be defined in a module. Each function has one or more arguments (however, technically all functions have only one argument; this is called [function currying][function-currying]). All arguments are statically-typed, and the return type can be explicitly declared, or often inferred by the compiler. Often you will need to import other modules to use their functions. This is done using the `import` keyword followed by the module you'd like to import. A common module to import is `Prelude` which supplies basic functions and operators such as addition (`+`) and subtraction (`-`).
```

Comment:
You probably don't want to mention function currying here, but mention that in `after.md`

Reply:
I think this block of text is too dense. It's explaining "must be in a module", functions have:
- arguments
- ...which are typed
- a return type
- ...which is inferred
- also imports
- ...and a common import
- ...and operators exist in that import.

I suggest breaking all these out in smaller bits, with examples.